src/lib/party.ts

```diff
@@ -492,6 +503,56 @@ type UnfinishedSignature = undefined | LazySignature | string;
 
 type LazyControl = Control | LazySignature | LazyProof;
 
+class Token {
```

Comment:
(no need to change it though if you feel the class has a use case)

Reply:
I'll leave it as a class for now since that was what the RFC suggested when we agreed upon everything. It's nice to have all these different properties lumped together in one object as well :D